VERSION

```diff
@@ -1 +1 @@
-0.4.4
+0.4.41
```

Comment:
Did you mean this version increase or this is just a remnant of your previous testing?

Reply:
I did mean to increase it, because I forgot when I changed public task methods in the ManualTest DSL resolution stuff.